packages/libs/react-components/src/components/Icons/Icon.stories.tsx

```diff
@@ -12,24 +13,36 @@ import React from 'react';
  * See https://storybook.js.org/docs/7.0/react/api/csf
  * to learn how to use render functions.
  */
-const meta: Meta<{ icon: string }> = {
+const meta: Meta<{ icon: string; color: string } & Icons.IIconProps> = {
   title: 'Icons',
   argTypes: {
     icon: {
       control: {
         type: 'text',
       },
     },
+    size: {
+      options: ['sm', 'md', 'lg'],
```

Comment:
nitpick: Would be better to get these from the variant object so if it ever changes this will automatically get updated